backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -65,14 +64,21 @@
             </div>
 
             <div class="col-12 col-xl-6">
-              <div class="field">
-                <%= label_tag :q_bill_address_firstname_start, t('spree.first_name_begins_with') %>
-                <%= f.text_field :bill_address_firstname_start, size: 25 %>
-              </div>
-              <div class="field">
-                <%= label_tag :q_bill_address_lastname_start, t('spree.last_name_begins_with') %>
-                <%= f.text_field :bill_address_lastname_start, size: 25%>
-              </div>
+              <% if Spree::Config[:show_deprecated_attributes] %>
+                <div class="field">
+                  <%= label_tag :q_bill_address_firstname_start, t('spree.first_name_begins_with') %>
+                  <%= f.text_field :bill_address_firstname_start, size: 25 %>
+                </div>
+                <div class="field">
+                  <%= label_tag :q_bill_address_lastname_start, t('spree.last_name_begins_with') %>
+                  <%= f.text_field :bill_address_lastname_start, size: 25 %>
+                </div>
+              <% else %>
+                <div class="field">
+                  <%= label_tag :q_bill_address_name_start, t('spree.name_begins_with') %>
```

Comment:
We are removing the ability to search by the initial part of the lastname here right? 😱

Reply:
Yes 😢 , I was thinking of switching the `starts with` searches to `contains` searches but it could have a big impact on db load and performances. The `LIKE` predicate used in `starts with` could apply the standard index,  but if we swtich to `contains` we would need a full text index. Full text indexes are different on many DBMS and I'm not sure if we can expect that they would be correctly configured.
Is there another way?

Reply:
Yes 😢 , I was thinking of switching the starts with searches to contains searches but it could have a big impact on db load and performances. The LIKE predicate used in starts with could apply the standard index, but if we swtich to contains we would need a full text index. Full text indexes are different on many DBMS and I'm not sure if we can expect that they would be correctly configured.
Is there another way?